app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` để làm gì e?

Reply:
Dạ "&." là Safe Navigation Operator đây là cách viết an toàn thông thường để đảm bảo không có các exception không mong muốn bắn ra khi object bị nil.

Reply:
Ngoài ra user&.authenticate(params[:session][:password]) là viết tắt của user&user.authenticate(params[:session][:password])